actions/rootio/v1/pkg/types.go/metadata.go

```diff
@@ -69,12 +69,16 @@ func RetrieveData() (*Metadata, error) {
 	if metadataURL == "" {
 		return nil, errors.New("unable to discover the metadata server from environment variable [MIRROR_HOST]")
 	}
+	port, exist := os.LookupEnv("ROOTIO_PORT")
```

Comment:
This port is for calls to the metadata service. Would you be open to changing the env var name to something that helps users understand its for the metadata call?